compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -107,6 +107,36 @@ pub fn expand_test_or_bench(
         return vec![];
     }
 
+    let not_testable_error = |item: Option<&ast::Item>| {
+        let diag = &cx.sess.parse_sess.span_diagnostic;
+        let msg = "the `#[test]` attribute may only be used on a non-associated function";
```

Comment:
Can you make this translatable? See #100717.

Reply:
Sorry, I'd prefer not to do this right now. The whole `test.rs` file hasn't been converted yet so I feel like that should happen all at once. Also there is some complexity with the error being downgraded to a lint in some cases.

I'm happy to do this if you really want, just not sure it's currently worth the effort.